### PR TITLE
feat(extractions): Design an UI to retrieve pending and active moon-mining extractions

### PR DESCRIPTION
**Pull request by warlof**
Provide a new tab called `Extractions` into corporation view which will show all active and pending extractions with moon location, structure and content.

Give the ability to send a moon probing report (based on in-game format stored with copy to clipboard button)

**Depends**: eveseat/eveapi#139
Closes eveseat/seat#379

image

image

### CODE REVIEW

**Comment by herpaderpaldent** [comment]
What do you think of adding the ability to manually add moons for corporations that you don't have a valid token on your instance?

The reason for this is: In goons we are renting the moons from alliance and that corporation we have no token access. Nevertheless being able to create manual moon-entries would be quite useful as the UI stands out!

File: src/Http/Controllers/Corporation/ExtractionController.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+class ExtractionController extends Controller
+{
+    /**
+     * @param int $corporation_id
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function getExtractions(int $corporation_id)
+    {
+        // retrieve any valid extraction for the current corporation
+        $extractions = CorporationIndustryMiningExtraction::with(
```
[the remark this answers is not in the document]
Comment (warlof) [reply src/Http/Controllers/Corporation/ExtractionController.php]: Is there a reason to use it somewhere else ? :)

[the remark this answers is not in the document]
Comment (leonjza) [reply src/Http/Controllers/Corporation/ExtractionController.php]: Yes. API, console and plugins can leverage services.

File: src/Http/Controllers/Corporation/ExtractionController.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+use Seat\Eveapi\Models\Universe\UniverseMoonContent;
+use Seat\Services\Repositories\Corporation\Extractions;
+use Seat\Web\Http\Controllers\Controller;
+use Seat\Web\Http\Validation\ProbeReport;
+
+/**
+ * Class ExtractionController.
+ * @package Seat\Web\Http\Controllers\Corporation
+ */
+class ExtractionController extends Controller
```
Comment (leonjza) [inline src/Http/Controllers/Corporation/ExtractionController.php]: Lets add some translations for these redirect messages?

**Comment by leonjza** [comment]
LGTM! Thanks :D

**Pull request merged**